grouper/fe/routes.py

```diff
@@ -25,6 +25,7 @@
     ),
     (r"/search", handlers.Search),
     (r"/users", handlers.UsersView),
+    (r"/users/public-keys", handlers.UsersPublicKey),
```

Comment:
It's specific to ssh keys but so is the rest of the endpoints that have 'public-key' in the url path. I'd expect any future upgrade to support other types of public keys should include expanding this endpoint to support it.